client/components/OrderHistory.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchOrders } from '../redux/orderHistory/orderHistory';
+
+class OrderHistory extends Component {
+  componentDidMount() {
+    this.props.fetchOrders();
+  }
+
+  render() {
+    const { orders } = this.props;
+    if (orders.length > 0) {
```

Comment:
don't need >0

Reply:
an empty array still return truthy so I believe we do need the > 0.